packages/bot-web-ui/src/components/dashboard/tutorial-tab/sidebar.tsx

```diff
@@ -1,52 +1,80 @@
 import React from 'react';
-import { Tabs, Tab, Icon } from '@deriv/components';
+import { Tabs, Icon } from '@deriv/components';
 import { connect } from 'Stores/connect';
 import RootStore from 'Stores/index';
```

Comment:
same here, can we put default imports at the top, before the named imports? @farabi-deriv

Reply:
done